dracut/parse-kickstart

```diff
@@ -521,7 +521,10 @@ def write_ifcfg(filename, ifcfg):
 
 def process_kickstart(ksfile):
     handler = DracutHandler()
-    handler.ksdevice = os.environ.get('ksdevice')
+    try:
+        handler.ksdevice = proc_cmdline['ksdevice']
+    except KeyError:
+        log.info("ksdevice argument is not available")
```

Comment:
Just for others to be on the same note, this patch completes https://github.com/rhinstaller/anaconda/commit/45a1ff563b56a7000ef05701f8ad4971891ab605,

I have a nitpick: I'd log this as debug (if logging at all). It is ok and even normal that ksdevice is not supplied (we even consider it  as deprecated) and there are other means to learn if it was supplied from dracut's journalctl, so as info the message feels a bit noisy to me here.

Other than that looks ok to me, thanks.